Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -100,20 +100,9 @@ @implementation Branch
 
 // PUBLIC CALLS
 
-+ (Branch *)getInstance:(NSString *)key {
-    [BNCPreferenceHelper setAppKey:key];
-    
-    if (!currInstance) {
-        [Branch initInstance];
-    }
-    
-    return currInstance;
-}
-
 + (Branch *)getInstance {
     if (!currInstance) {
```

Comment:
can we add some key validation and warning code? I want to make sure forgetting the plist step is really obvious and the the logs will tell them what to do